src/components/scenes/WalletListScene.js

```diff
@@ -10,7 +10,7 @@ import { updateActiveWalletsOrder } from '../../actions/WalletListActions.js'
 import XPubModal from '../../connectors/XPubModalConnector.js'
 import s from '../../locales/strings.js'
 import { getIsAccountBalanceVisible } from '../../modules/Settings/selectors.js'
-import { getActiveWalletIds, getWalletLoadingPercent } from '../../modules/UI/selectors.js'
+import { getWalletLoadingPercent } from '../../modules/UI/selectors.js'
```

Comment:
could we also remove this? and pass `state.ui.wallets.walletLoadingProgress` instead of state?

Reply:
Actually, I have a commit that does this, but I didn't submit it with the PR because I thought it was out-of-scope. If you want, I will throw that commit on the end of this PR so you can look at it too.

Reply:
ok, let's leave it in next PR

Reply:
I went ahead and added it to this PR, since the commit was ready to go.